Dz3/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection.Metadata.Ecma335;
+using System.Text.RegularExpressions;
+
+namespace DZ2DZ3
+{
+    public class Program
+    {
+        public enum Gender
+        {
+            male,
+            female,
+            undefined
+        }
+        static void Main(string[] args)
+        {
+            User user1 = new User();
+            // user1.Gender = Gender.Male;
+            try
+            {
+                while (true)
+                {
+                    Console.WriteLine(" Pleace input your name");
+                    user1.FirstName = Console.ReadLine();
+
+                    if (Regex.IsMatch(user1.FirstName, "^[a-zA-Z]+$"))
```

Comment:
And if I make a name with 10000000000 symbols???

Reply:
it doesn`t matter, coz this is your name and include only letters